explore/src/main/scala/explore/targeteditor/SpectralDefinitionEditor.scala

```diff
@@ -195,6 +194,14 @@ sealed abstract class SpectralDefinitionEditorBuilder[
         )
 
       React.Fragment(
+        props.catalogInfo.flatMap(ci =>
+          ci.objectType.map(ot =>
+            React.Fragment(
+              FormLabel(htmlFor = "catalogInfo".refined)(ci.catalog.shortName),
```

Comment:
What if we get this from an imported target?

Reply:
I see it comes from the `Display` instance

Reply:
Right. But I'll add a label for the imported case.